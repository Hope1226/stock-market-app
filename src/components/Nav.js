import React from 'react';
import { Link } from 'react-router-dom';

const Nav = () => (
  <nav>
    <ul>
      <Link to="/popular" className="navLink">
        <li>Popular</li>
      </Link>
      <Link to="/gainers" className="navLink">
        <li>Gainers</li>
      </Link>
      <Link to="/losers" className="navLink">
        <li>Losers</li>
      </Link>
    </ul>
  </nav>
);

export default Nav;
