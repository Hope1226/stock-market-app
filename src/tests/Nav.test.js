import React from "react";
import renderer from 'react-test-renderer';
import { MemoryRouter } from "react-router-dom";
import Nav from '../components/Nav';

describe('Should load the navigation', ()=> {
  test('and should match the snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Nav />
      </MemoryRouter>,
    ).toJSON;

    expect(tree).toMatchSnapshot();
  });
});