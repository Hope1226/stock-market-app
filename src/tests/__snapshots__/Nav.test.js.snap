// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should load the navigation and should match the snapshot 1`] = `[Function]`;
